src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: rgb(170, 204, 221);
```

Comment:
Try to use HEX instead of RGB

Reply:
Hello, Yehor Korenkov sad "Hello, when deploying, the compiled file is automatically substituted, so I think you can skip this comment".